src/ch02-00-guessing-game-tutorial.md

```diff
@@ -122,9 +131,9 @@ let mut banana = 5; // 可變的
 
 `&` 說明此引數是個**引用（reference）**，這讓程式中的多個部分可以取得此資料內容，但不需要每次都得複製資料到記憶體中。引用是個複雜的概念，而 Rust 其中一項主要優勢就是能夠輕鬆又安全地使用引用。你現在還不用知道一堆細節才能完成程式。現在你只需要知道引用和變數一樣，預設都是不可變的。因此你必須寫 `&mut guess` 而不是 `&guess` 才能讓它成為可變的。（第四章會再全面詳細解釋引用。）
 
-### 使用 `Result` 型別處理可能的錯誤
+### 使用 `Result` 處理可能的錯誤
```

Comment:
I didn't add this because some titles have many historical ones. I assume it's for compatibility of external link.
We couldn't track all the them and I already make sure the links are valid within the book. So I feel like we can pass this.